gpios.md: add known issue section - apuled driver conflict (#162)
pcengines/apu2-documentation: docs/gpios.md

> Important: when exporting, pin changes its state to input and low state. In
> BIOS these pins are configured as output high. Be sure to configure them to
> output first before using them.

## Known issue

Since coreboot release *v4.10.0.0* there is conflict with **apuled** driver in
FreeBSD. It is because BIOS reserves resources for GPIOs with ACPI controller

> support. Then somehow, FreeBSD cannot reserve the memory for native driver

Reviewer: ```suggestion
support. As a result, FreeBSD cannot reserve the memory for native driver
```
